ods_ci/run_interop.sh

```diff
@@ -65,14 +84,11 @@ if [ $retVal -ne 0 ]; then
     exit "$retVal"
 fi
 
-if [[ -z "${TEST_SUITE}" ]]; then
-  echo "Define TEST_SUITE"
-  exit 1
-fi
+if [[ ${TEST_SUITE} == "PreUpgrade" ]]; then
+  echo "Save test config file..."
+  cp test-variables.yml ${SHARED_DIR}/test-variables.yml
```

Comment:
I think it will be better to use the defined variable TEST_VARIABLES_FILE instead of writing the file name like:
```suggestion
  cp ${TEST_VARIABLES_FILE} ${SHARED_DIR}/${TEST_VARIABLES_FILE}
```

Reply:
updated